src/dune_pkg/fetch.mli

```diff
@@ -17,10 +17,12 @@ val fetch
   -> checksum:Checksum.t option
   -> target:Path.t
   -> OpamUrl.t
+  -> url_loc:Loc.t
```

Comment:
When it's not clear what which argument the location pertains to, dune's convention is usually to tuple it. So in this case, it should be `(Loc.t * OpamUrl.t)`.